doc/portfile.7

```diff
@@ -2696,7 +2975,7 @@ Display a
 to the user, at various different levels.
 .br
 .Sy Example:
-.Dl ui_msg \*qAdd each user to the system using the clamav command\*q
+.Dl ui_msg \*qThe MySQL client has been installed.\*q
```

Comment:
This is not really what `ui_*` should be used for, but the previous example was not that good either (because we have `notes` for that now). In fact, I cannot even think of a valid use case for `ui_msg` except for debugging purposes...

Reply:
As I've mentioned in the commit message, I didn't change the contents of the document – to do that, my knowledge is simply too limited.

Reply:
I don't have time to do such a conversion, sorry.  I suggest to use mandoc's ability to convert [mdoc to markdown](https://undeadly.org/cgi?action=article&sid=20170304230520) and then [pandoc](http://pandoc.org) to convert markdown to AsciiDoc – the more precise the mdoc sources are tagged, the better the conversion, BTW.  So maybe my work wasn't completely useless...

Note that one of the authors of mandoc, Ingo Schwarze, is very helpful; he can certainly assist in case problems arise during the conversion.  Maybe you can even persuade him to make mandoc directly emit AsciiDoc, assuming that you give him a helping hand.

Reply:
Sorry to reopen this, but I pointed at this hunk particularly because the content did in fact change. The question was whether this happened intentionally, as I did not expect it in this commit.

Your changes definitely are useful and I am convinced we should merge this. I just wanted to hint at the need for a  conversion to AsciiDoc to match the other man pages. However, I fully understand if this is out of your scope. For the other man pages (such as `port.1` to `port.1.txt`) I used a manual approach and rewrote them in AsciiDoc from the rendered man output, as no automatic tool produced satisfactory results (but note this was like 10 years ago).

Reply:
Since mandoc is semantic markup, scripting the conversion might not be too bad.

Reply:
@raimue I wanted to shorten the text to avoid overlong lines as much as possible if the manpage gets formatted with `groff -ww -Tpdf -mdoc -rS12 portfile.7 > portfile.7.pdf` – having too small a body font makes the long lines of the text hard to read.  I did a grep for `ui_msg` in the 'macports_ports' git repository and arbitrarily took a short string.

Reply:
`notes`, which prints a message after installation, has indeed largely made `ui_msg` unnecessary. There are a few ports that use `ui_msg` to print a message during installation, such as to notify the user that a fetch or a build may take longer than usual (bazel, InsightToolkit, inkscape-devel); I'd change the example to something like that.

```
.Dl ui_msg \*qThis build may take a very long time\*q
```

Reply:
Done.